### PR TITLE
Birds can fly from both directions

### PR DESCRIPTION
**Pull request by PontusNyberg**
* added gitignore to not accidently add node_modules or package-lock.json
* Made a function to generate if bird come from left or right

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -58,8 +58,18 @@ function love.update(dt)
...
-            table.insert(birbs, birb)
+            generateBirb()
+        end
+
+        function generateBirb()
+            -- left 1, right 2
+            local randomSide = math.random(2);
+
+            if(randomSide == 2) then
+                table.insert(birbs, Bird.new(1100, math.random(0, 350)))
```
[the remark this answers is not in the document]
Comment (phlogios) [reply main.lua]: But if the screen size is changed, this value needs to be changed manually to fit the new screen size.

**Comment by phlogios** [comment]
This project does not use Node or NPM and will therefore never generate node_modules folders or package.json files.

**Comment by PontusNyberg** [comment]
Hmm, wonder why I got those folders then :p probably need to check that then because all I did was clone the project and open the folder in visual studio code. Could it add those folders because I have some addons installed maybe?

**Comment by phlogios** [comment]
That's probably it!

File: main.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (honeyp0t) [reply main.lua]: I would like to see 1100 replaced with love.graphics.getWidth() 💯

**Comment by PontusNyberg** [comment]
Closing this and make a new one with .gitignore removed and dynamic width.

**Pull request closed without merging**